src/test/java/hudson/security/docker/MultiServerTest.java

```diff
@@ -23,12 +24,19 @@
 import static org.hamcrest.core.StringContains.containsString;
 import static org.junit.Assert.assertEquals;
 import static org.junit.Assert.assertThat;
+import static org.junit.Assume.assumeFalse;
+import org.junit.BeforeClass;
 
 /**
  * Tests connecting to two different servers
  */
 @LDAPTestConfiguration
 public class MultiServerTest {
+
+    @BeforeClass public static void linuxOnly() {
```

Comment:
this makes me sad as it can do it - and most devs on windows can also do it, but now the tests will all be skipped.

can you check for a `JENKINS_URL` or `BUILD_URL` in the environment so this is not skipped if docker is available and we are running in CI?

Reply:
unrelated but maybe passing  `--platform  linux/amd64` would help (or at least we could specify the requirement in the `DockerContainer` such that it could test that the platform was available locally?

Reply:
Please comment in #47 instead.